lib/api/model/model.dart

```diff
@@ -308,6 +308,14 @@ enum UserRole{
 /// in <https://zulip.com/api/register-queue>.
 @JsonSerializable(fieldRename: FieldRename.snake)
 class ZulipStream {
+  // When adding a field to this class:
```

Comment:
commit message nit: Let's try to use more brief wording, say:
```
model [nfc]: Clarify how to add updatable fields to `ZulipStream`
```
or
```
model [nfc]: Clarify when to add non-final fields to `ZulipStream`
```
Also, I believe this is closely related to the previous commit
[api: Add stream/update event](https://github.com/zulip/zulip-flutter/pull/880/commits/93ce9c5c4e5c7776ca2cb9634866f6f23e63b448)
So, most probably this will get squashed into it during maintainer or integration review.